app/indexers/plum_jsonld_converter.rb

```diff
@@ -255,4 +256,16 @@ def physical_number
     def box_physical_number
       box["box_number"]
     end
+
+    def collections
+      eligible_collections.map do |member|
+        member["title"]
+      end
+    end
+
+    def eligible_collections
+      Array.wrap(json["memberOf"]).select do |member|
+        member["barcode"].blank? && member["title"] != "Latin American Ephemera"
```

Comment:
Can you add a comment explaining these checks?

Reply:
Done.